mslib/mswms/mss_plot_driver.py

```diff
@@ -358,7 +358,7 @@ def set_plot_parameters(self, plot_object=None, vsec_path=None,
                             vsec_numlabels=10,
                             init_time=None, valid_time=None, style=None,
                             bbox=None, figsize=(800, 600), noframe=False,
-                            show=False, transparent=False,
+                            show=False, transparent=False, one_dimensional=False,
```

Comment:
Does it make sense to add the functionality here? It seems like a common base class or delegation might be more elegant...